site/content/notes/Nx.md

```diff
@@ -0,0 +1,225 @@
+# Nx
+
+## Applications and libraries
```

Comment:
This whole document is **excellent** and a super useful summary. 👏👏 It also illustrates the kind of "document/braindump" as learning approach that i think is great.